api/src/uvisor-input.S

```diff
@@ -114,6 +114,9 @@ uvisor_config:
     /* Debug driver pointer */
     .long __uvisor_debug_driver
 
+    /* Stack limit for MSPLIM_NS */
+    .long __uvisor_stack_limit
+
```

Comment:
i see you are missing the assignment here as you have assigned the value to the `__uvisor_stack_limit` from the mbed-os linker script.
Please consider assigning __StackLimit__ symbol directly without intermediate variable and without modifying mbed-os sources.